packages/amp/src/processors/removeProhibitedTags.ts

```diff
@@ -0,0 +1,17 @@
+import { Processor, Element } from "@frontity/html2react/types";
+import { Packages } from "../../types";
+
+export const removeProhibitedTags: Processor<Element, Packages> = {
+  name: "amp: removeProhibitedTags",
```

Comment:
I guess this processor should have a rather low priority in case other processors are adding new nodes with these tags, shouldn't it?

What about `priority: 20`? 
```suggestion
  name: "amp: removeProhibitedTags",
  priority: 20,
```